audiocall/src/main/java/com/voximplant/demos/kotlin/audio_call/services/AudioCallManagerDefault.kt

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2011 - 2024, Zingaya, Inc. All rights reserved.
+ */
+
+package com.voximplant.demos.kotlin.audio_call.services
+
+import android.content.Context
+import com.voximplant.demos.kotlin.utils.*
+import com.voximplant.sdk.call.*
+import com.voximplant.sdk.client.IClient
+import java.util.*
+
+class AudioCallManagerDefault(
+    context: Context,
+    client: IClient,
+) : AudioCallManager(context, client) {
+
+    override fun onIncomingCall(call: ICall, video: Boolean, headers: Map<String?, String?>?) {
+        super.onIncomingCall(call, video, headers)
+        showIncomingCallUI()
```

Comment:
AudioCallManager.onIncomingCall can reject the call if there is already one. 

In this case we should not show incoming call UI. The check for `callExists` in showIncomingCallUI here does not work in this case

Reply:
Updated